src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -285,10 +287,18 @@ impl StaticHeader {
                             (Some(loc_1), Some(loc_2)) => {
                                 if loc_1 == loc_2 {
                                     Ok(Some(loc_1))
+                                } else if loc_1.initialization_time == loc_2.initialization_time {
+                                    // Inexplicable disagreement among static headers
+                                    let err_str = "Appeared to be a Stratis device, but signature blocks disagree.";
+                                    Err(StratisError::Engine(ErrorEnum::Invalid, err_str.into()))
```

Comment:
IMHO this would be extremely unlikely, but if it did happen I would like to have the contents of each super blocks dumped to the logs.